integration-tests/src/test/java/eu/tsystems/mms/tic/testframework/test/utils/FileDownloaderTest.java

```diff
@@ -106,6 +109,16 @@ public void test03_downloadFileToLongLocation() throws IOException {
         Assert.assertTrue(file.exists(), "File was downloaded correctly.");
     }
 
+    @Test
+    public void test_ContentDisposition_parsing() {
```

Comment:
This testcase does not test a behaviour of Testerra, right? It's just using the same logic/implementation that Testerra will use under the hood.

So what's the matter of executing it, when not testing the actual behaviour of Testerra?

Reply:
Because the already implemented test is wrong. It actually doesn't test reading the file name from content disposition header, because the file from Wikipedia doesn't provide it. And I did not found a public available picture with content-disposition header to fix the test.

Reply:
Okay, that's the same thoughts I had on that.
Is it possible to provide such a file downlaod in test-sites that are used for integration-testing?

Reply:
Only when I'm able to provide the header information, so this is just much more effort than just use a public file that provides it.